sdk/communication/Azure.Communication.Chat/tests/Infrastructure/ChatRecordedTestSanitizer.cs

```diff
@@ -14,13 +12,5 @@ public class ChatRecordedTestSanitizer : CommunicationRecordedTestSanitizer
 
         public ChatRecordedTestSanitizer() : base()
```

Comment:
Remove the sanitizer type and call AddJsonPathSanitizer in the test base?

Reply:
Once the migration is complete, all sanitizers will be removed so I'll remove it at that point.